VespaGAS/Managers/Store/VespaStore.swift

```diff
@@ -0,0 +1,9 @@
+import ReSwift
+
```

Comment:
The store is responsible to dispatch actions to the reducer and sending the new state to his subscribers.